TESTING.rst

```diff
@@ -278,7 +278,7 @@ Local and Remote Debugging
 ==========================
 
 One of the great benefits of using the local virtualenv and Breeze is an option to run
-local debugging in your IDE graphical interface. You can also use ``ipdb``
+local debugging in `your IDE graphical interface <BREEZE.rst#using-your-host-ide>`__. You can also use ``ipdb``
 if you prefer `console debugging <#breeze-debugging-with-ipdb>`__.
```

Comment:
I tried to find this broken link with `grep -R 'ipdb' .` but I could not find anything relevant. Was this rewritten into something else? Or should I remove the link if this was removed all together?

Reply:
I can't find this section either.

Reply:
@potiuk Do you know where to find these documentation?

Reply:
Fixed it in https://github.com/apache/airflow/pull/7104